swift/Sources/FlatBuffers/Struct.swift

```diff
@@ -28,15 +28,15 @@ public struct Struct {
   /// Hosting Bytebuffer
   public private(set) var bb: ByteBuffer
   /// Current position of the struct
-  public private(set) var postion: Int32
+  public private(set) var position: Int32
 
   /// Initializer for a mutable flatbuffers struct
   /// - Parameters:
   ///   - bb: Current hosting Bytebuffer
   ///   - position: Current position for the struct in the ByteBuffer
   public init(bb: ByteBuffer, position: Int32 = 0) {
     self.bb = bb
-    postion = position
+    self.position = position
```

Comment:
I was told this is the way to disambiguate between the member and the argument -- I believe I've properly applied these changes (they were hand applied).